modules/deploy/pages/deployment-option/cloud/monitor-cloud.adoc

```diff
@@ -0,0 +1,71 @@
+= Monitor Redpanda Cloud
+:description: Learn how to integrate the Redpanda Cloud metrics endpoint to monitor the health of your Dedicated or BYOC clusters.
+:page-cloud: true
+:env-cloud: true
+
+You can configure monitoring on your Redpanda Dedicated or BYOC cluster to maintain system health and optimize performance. Redpanda exports metrics from the xref:reference:public-metrics-reference.adoc[`/public_metrics`] Prometheus endpoint through the Admin API port 9644. 
+
+== Configure monitoring
+
+You can monitor Redpanda with Prometheus or with any other monitoring and alerting tool, such as Datadog, New Relic, Elastic Cloud, Google, or Azure.
+
+[NOTE]
+====
+To maximize performance, Redpanda exports some metrics only when the underlying feature is in use. For example, a metric for consumer groups, xref:reference:public-metrics-reference.adoc#redpanda_kafka_consumer_group_committed_offset[`redpanda_kafka_consumer_group_committed_offset`], is only exported when groups are registered.
+====
+
+=== Configure Prometheus
+
+To monitor a Dedicated or BYOC cluster in https://prometheus.io/[Prometheus^]:
+
+. On the Redpanda Cloud *Overview* page for your cluster, under *How to connect*, click the *Prometheus* tab. 
+
+. Click the copy icon for *Prometheus YAML* to copy the contents to your clipboard. 
++
+image::shared:cloud_metrics.png[How to connect screenshot]
++
+The YAML contains the Prometheus scrape target configuration, as well as authentication, for the cluster.
++
+```
+- job_name: redpandaCloud-sample
+  static_configs:
+    - targets:
+        - console-321f60a4.cn6b1srsj6j1mdqdhil0.fmc.ppd.cloud.redpanda.com
```

Comment:
Since this is an example, we should scrub this so it doesn't have a real hostname. How we did this below in the Datadog example section is probably good (e.g., console-<id>.<clusterid>.fmc.blahblah)